services/database-analyzer.js

```diff
@@ -127,6 +127,20 @@ function DatabaseAnalyzer(databaseConnection, config) {
     tableForeignKeysAnalyzer = new TableForeignKeysAnalyzer(databaseConnection, config);
     columnTypeGetter = new ColumnTypeGetter(databaseConnection, config.dbSchema || 'public');
 
+    if (config.dbSchema) {
+      const schemaExists = await queryInterface.sequelize
+        .query(
+          `SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = '${config.dbSchema}';`,
```

Comment:
I don't actually know sequelize but it seems this is vulnerable to SQL injection. You could use Replacements (https://sequelize.org/master/manual/raw-queries.html#replacements). Still, I know the dev who use this query should not be the one that want to run an injection (and she actually _could_ by other ways), but I think it's better to protect SQL injection everywhere.